addon/manifests/chart/templates/deployment.yaml

```diff
@@ -48,7 +64,15 @@ spec:
           value: {{ .Values.global.proxyConfig.NO_PROXY }}
         {{- end }}
         resources:
-          {{- toYaml .Values.resources | nindent 10 }}
+          requests:
+            memory: 128Mi
```

Comment:
Can we also allow to customize the request. This may also cause problems in some scenarios.

Reply:
@jlpadilla Could you please explain the scenario - Request should be the minimum that we need to run the pod . And we want our users to get that.

Reply:
`request` is the minimum amount or memory that kubernetes will **guarantee**.  If the pod is using more than the request, the kubernetes scheduler may kill the pod to assign it to another workload with a higher priority.

In some large clusters, users may need to change the request to ensure that the pod gets the necessary resources.

Reply:
request should be set to the real avg usage of your pod, if you declare 128Mi but you are using 1Gi the scheduler won't be able to assign properly resources and you can cause evictions. So this should be definitely customizable

Reply:
Added support for request memory. @andreadecorte @jlpadilla

Reply:
thanks